packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -77,8 +77,8 @@ export const ownPropTypes = {
 const defaultProps = {
   extendedLoad: true,
   enableAddButton: false,
-  enableUserProfile: true,
-  enableSpaceListFilter: true,
+  enableUserProfile: false,
+  enableSpaceListFilter: false,
```

Comment:
Why are we changing the default props? We've released these already and this breaks the existing usage.

Reply:
![screen shot 2019-03-07 at 9 32 13 am](https://user-images.githubusercontent.com/14226615/53976465-f087bb80-40bb-11e9-9002-4d58bccc583a.png)
The default props are false. I understand that this was merged before. But it contradicts with what we have in mind for design.

Reply:
Default props for the widget and default props for the demo are not the same. Check out the documented defaults: https://github.com/webex/react-ciscospark/tree/master/packages/node_modules/@ciscospark/widget-recents#configuration

Reply:
so what are default props for widget? if thats the case then we need to stay consistent with the widget demo. As far as I remember, we decided not to show any header unless the user wanted to

Reply:
addressed